lms/envs/common.py

```diff
@@ -2191,17 +2191,24 @@
 ECOMMERCE_API_TIMEOUT = 5
 
 # PROFILE IMAGE CONFIG
-# TODO: add these settings to aws.py as well
-PROFILE_IMAGE_BACKEND = 'django.core.files.storage.FileSystemStorage'
-# PROFILE_IMAGE_DOMAIN points to the domain from which we serve image
-# files from.  When this is '/', it refers to the same domain as the
-# app server.  If serving from a different domain, specify that here
-# i.e. 'http://www.example-image-server.com/'
-PROFILE_IMAGE_DOMAIN = '/'
-PROFILE_IMAGE_URL_PATH = 'media/profile_images/'
+# WARNING: Certain django storage backends do not support atomic
+# file overwrites (including the default, OverwriteStorage) - instead
+# there are separate calls to delete and then write a new file in the
+# storage backend.  This introduces the risk of a race condition
+# occurring when a user uploads a new profile image to replace an
+# earlier one (the file will temporarily be deleted).
+PROFILE_IMAGE_BACKEND_CONFIG = {
+    'storage_class': 'storages.backends.overwrite.OverwriteStorage',
+    'storage_kwargs': {
+        'location': '/edx/var/edxapp/uploads/profile_images/',
+        'base_url': '/media/profile_images/',
```

Comment:
I vote we switch this to `/edx/var/edxapp/media/profile-images`. `uploads` already exists and I have no idea what uses it.